Components/BasketDetails.js

```diff
@@ -0,0 +1,51 @@
+import React, {Component} from 'react';
+import {StyleSheet, Platform, View, Text, FlatList, TouchableOpacity} from 'react-native';
+import baskets from '../Helper/basketData'
+import ProductItem from './ProductItem'
+import {getProductInfoFromApi} from "../API/OFFApi";
+
+class BasketDetails extends Component {
+
+    constructor(props) {
+        super(props);
+        this.basketId = this._getBasketFromId(this.props.navigation.getParam('basketId'));
```

Comment:
Je ne comprends pas très bien, j'ai l'impression que `_getBasketFromId` ne renvoie rien donc qu'est-ce qui est assigné à `this.basketId` ?

Reply:
En effet la fonction est pas construite. Actuellement _getBasketFromId va chercher le panier correspondant au basketid dans la bdd et actualiser l'attribut this.basket.
Je vais faire en sorte que la fonction renvoie quelque chose

Reply:
J'ai finalement utilisé un shéma similaire à Product.js